server/restivus-swagger.js

```diff
@@ -11,7 +11,7 @@ Restivus.prototype.addSwagger = function(path) {
   restivus.addRoute(path, {authRequired: false}, {
     get: function () {
       // Check if swagger meta configuration exists
-      if(config.swagger) {
+      if(config.swagger !== undefined) {
```

Comment:
Consider using `_.has()` here to improve readability, and further explore the underscore API.

E.g.

```js
if ( _.has(config, "swagger") ) {

}
```